doc/users/next_whats_new/colormaps.rst

```diff
@@ -1,8 +1,9 @@
-Colormap registry
-------------------
+Colormap registry (experimental)
+--------------------------------
 
 Colormaps are now managed via `matplotlib.colormaps`, which is a
-`.ColormapRegistry`.
+`.ColormapRegistry`. This API is experimental and may still change in future
+versions.
```

Comment:
Could we add a note somewhere here encouraging users to leave feedback on the new API via. opening an issue? I think that's an important part of "experimental", and would encourage more users to try it out.

Reply:
I think you haven't been on the last dev call. We had a discussion there whether this should be experimental or not. I feel very confident on the API and there is not much design space anyway (It's a read-only dict plus a register method and the name `colormaps` is quite canonical as well). The decision for "experimental" was more a "let's be cautious just in case".

As we essentially do not have any uncertainties in the API design, I don't want to make a big fuzz or encourage discussions. If somebody nevertheless sees a point, they can open an issue as usual. I don't think this needs special mention. I also don't follow the argument that "you can give feedback if you try this out" would be a strong motivation for trying out.

Reply:
> As we essentially do not have any uncertainties in the API design

Sorry to potentially cover ground that has already been covered elsewhere, but if there's no API uncertainty then why is it experimental API? I think "experimental" needs clarifying with a few more words - by itself it could mean anything from "we're trying it out and might just get rid of it later" to "it's pretty much final, but we reserve the right to make changes in response to user feedback".

> I also don't follow the argument that "you can give feedback if you try this out" would be a strong motivation for trying out

I think empowering users to have an opinion on experimental API fits in with the ethos of a community driven open source project.